pkg/setting/setting_secrets_manager.go

```diff
@@ -35,6 +47,12 @@ func (cfg *Cfg) readSecretsManagerSettings() {
 	cfg.SecretsManagement.GrpcServerAddress = valueAsString(secretsMgmt, "grpc_server_address", "")
 	cfg.SecretsManagement.GrpcGrafanaServiceName = valueAsString(secretsMgmt, "grpc_grafana_service_name", "")
 
+	cfg.SecretsManagement.GCWorkerEnabled = secretsMgmt.Key("gc_worker_enabled").MustBool(true)
+	cfg.SecretsManagement.GCWorkerMaxBatchSize = uint16(secretsMgmt.Key("gc_worker_batch_size").MustUint(16))
+	cfg.SecretsManagement.GCWorkerMaxConcurrentCleanups = uint16(secretsMgmt.Key("gc_worker_max_concurrency").MustUint(16))
+	cfg.SecretsManagement.GCWorkerPollInterval = secretsMgmt.Key("gc_worker_batch_size").MustDuration(250 * time.Millisecond)
```

Comment:
Q: Do we want to be this aggressive with GCing secrets? Would every 1 minute not be too often you think?